nextcord/application_command.py

```diff
@@ -835,7 +835,11 @@ async def can_run(self, interaction: Interaction) -> bool:
         return True
 
     async def invoke_callback_with_hooks(
-        self, state: ConnectionState, interaction: Interaction, *args, **kwargs
+        self,
+        state: ConnectionState,
+        interaction: Interaction,
+        args: List[Any] = [],
+        kwargs: Dict[str, Any] = {},
```

Comment:
Is there a way we can avoid using mutable defaults here?

N.b. I'm aware they are not mutated within this method but it'd still be a good idea for the future if changes were to occur in this method which mutated them

Reply:
Wdym "mutable defaults"
Like setting the defaults to an empty list and an empty dict?

Reply:
Correct

Reference [link](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) explaining it a bit better then I can

Reply:
Ohhhhhhh I see now

Reply:
1de830e